tools/manually-handling-sharding-ddl-locks.md

```diff
@@ -0,0 +1,415 @@
+- `force-remove`: flag; boolean; `--force-remove`; optional; if not being specified, remove the lock information only when the owner succeeds to execute the DDL; if being specified, compulsorily remove the lock information even though the owner fails to execute the DDL (after doing this you will not be able to query or operate on the lock again)
+- `lock-ID`: non-flag; string; optional; specify the ID of the DDL lock which needs to be unlocked (the `ID` in the result of `show-ddl-locks`)
+
+##### Example of results
+
+```bash
+» unlock-ddl-lock test-`shard_db`.`shard_table`
+{
+    "result": true,                                        # show if the unlocking process succeeds
+    "msg": "",                                             # show the reason why the unlocking process fails
+    "workers": [                                           # the result list of the DDL execution/skipping operation of each DM-worker
+        {
+            "result": true,                                # show if the DDL execution/skipping operation of the DM-worker succeeds
+            "worker": "127.0.0.1:8262",                    # the address of the DM-worker (DM-worker ID)
+            "msg": ""                                      # show the reason why the DDL execution/skipping operation of the DM-worker fails
+        }
+    ]
+}
+```
+
+#### break-ddl-lock
+
+Actively ask the DM-worker to compulsorily break the DDL lock that is waiting to be unlocked, for example, ask the DM-worker to execute/skip the DDL; remove the DDL lock information on the DM-worker.
+
+##### Command usage
+
+```bash
+break-ddl-lock <--worker=127.0.0.1:8262> [--remove-id] [--exec] [--skip] <task-name>
+```
+
+##### Variables description
+
+- `worker`: flag; string; `--worker`; not optional; specify the DM-worker which needs to execute the breaking operation
+- `remove-id`: flag; string; `--remove-id`; optional; if being specified, it should be the ID of some DDL lock; if not being specified, remove the corresponding DDL lock information only when the breaking operation succeeds; if being specified, compulsorily remove the DDL lock information 
+- `exec`: flag; boolean; `--exec`; optional; cannot be specified simultaneously with the `--skip` parameter; if being specified, ask the DM-worker to execute the corresponding DDL of the lock 
+- `skip`: flag; boolean; `--skip`; optional; cannot be specified simultaneously with the `--exec` parameter; if being specified, ask the DM-worker to skip the corresponding DDL of the lock 
+- `task-name`: non-flag; string; not optional; specify the name of the task containing the lock that is going to execute the breaking operation (you can check if a task contains the lock via [query-status](../task-handling/query-status.md))
+
+##### Example of results
+
+```bash
+» break-ddl-lock -w 127.0.0.1:8262 --exec test
+{
+    "result": true,                                        # check if the lock breaking operation succeeds
```

Comment:
`show if the lock breaking operation succeeds`?

Reply:
Can I say "the result of the lock breaking operation"?

Reply:
OK.